src/main/requests/page/user_long_lived_token/GetUserLongLivedTokenResponse.ts

```diff
@@ -0,0 +1,19 @@
+import { AbstractResponse } from '../../AbstractResponse';
+import { UserLongLivedTokenData } from '../../data/UserLongLivedTokenData';
+
+/**
+ * Represents a response from the {@link GetUserLongLivedTokenRequest}"
```

Comment:
Just for consistency with what I've been putting on other classes:

```suggestion
 * Represents a response from a Get UserLong Lived Token Request
```

Reply:
I had some time so I made the commit with the changes I request, pretty normal stuff. Also updated the readme. Thanks again for this 👍